### DISCUSSION TITLE
Pushing to non-bare repo

### DISCUSSION THREAD

**Discussion by cmyplay** [libgit2/libgit2#6913] [Q&A]
Git makes pushing to a non-bare repo possible using `config receive.denyCurrentBranch updateInstead`
Are there plans to include this as a feature in LG2? If not is there another way to achieve this?

**Comment by ethomson** [comment]
I'm not sure that I understand; isn't that just a config setting for the server side (the receiver)? Since we don't have a server (yet), I don't think that this is something that we could support. But maybe I misunderstand.
